pkgs/applications/graphics/write_stylus/default.nix

```diff
@@ -16,6 +17,12 @@ stdenv.mkDerivation rec {
     url = "http://www.styluslabs.com/write/write${version}.tar.gz";
     sha256 = "1p6glp4vdpwl8hmhypayc4cvs3j9jfmjfhhrgqm2xkgl5bfbv2qd";
   };
+
+  icon = fetchurl {
+    url = "https://oyra.eu/write/icon.tar.gz";
```

Comment:
What is the license of this file. Is it your own creation or what is the origin?

Reply:
Remember not to change/remove this file.

Reply:
Sure.

Reply:
> Remember not to change/remove this file.

tl;dr
the file was removed and the package is uninstallable now...

Reply:
True, it's on my todo list but i haven't found the time yet.

Reply:
#60090